bgpd/bgp_route.c

```diff
@@ -8132,8 +8135,19 @@ int bgp_show_table_rd(struct vty *vty, struct bgp *bgp, safi_t safi,
 				       output_arg, use_json,
 				       rd, next == NULL,
 				       &output_cum, &total_cum);
+			if (next == NULL)
```

Comment:
This test has me confused.  Why would I ever want to turn off the display of prefixes if I've displayed one table but the next one is NULL?